crates/net/network/src/network.rs

```diff
@@ -55,6 +55,7 @@ impl NetworkHandle {
             network_mode,
             bandwidth_meter,
             is_syncing: Arc::new(AtomicBool::new(false)),
+            initial_sync_done: Arc::new(AtomicBool::new(false)),
```

Comment:
we could put both states in a single AtomicU8, but perhaps the added complexity is not worth it since we only ever use relaxed.
I'd be okay with both solutions

Reply:
yeah for sure - I agree in that this may be a bit more naive, but it's cheap for us to use another AtomicBool and we get the added benefit of readability/simplicity